api/admin/controller.py

```diff
@@ -168,6 +170,24 @@ def details(self, data_source, identifier):
         return entry_response(
             AcquisitionFeed.single_entry(self._db, work, annotator)
         )
+        
+    def complaints(self, data_source, identifier):
+        """Return detailed complaint information for admins."""
+        
+        pool = self.load_licensepool(data_source, identifier)
+        if isinstance(pool, ProblemDetail):
+            return pool
+        work = pool.work
+        counter = self._count_complaints_for_work(work)
+        response = dict({
+            "book": { 
```

Comment:
"work" is no good here because a Work is a specific thing in this system which is not identified by data source + identifier. Either Edition or LicensePool can be identified by data source + identifier. I would maybe call it 'edition'.